### ISSUE TITLE
Cookies page - accessibility fixes

### ISSUE THREAD

**Issue by mitz-lad** [dwp/design-system#535]
### Accessibility fixes
- First example code on cookies page has wrong content in `<legend>` tags for the Analytics cookies radio buttons - replace 'Can we use [category] cookies?' to 'Can we use analytics cookies?' 
- Second example code on cookies page has wrong content in `<legend>` tags for the Analytics cookies radio buttons - replace 'Can we use [category] cookies?' to 'Can we use analytics cookies?'

### Other findings
- Copy code buttons on page does not work
- Example 1 and Example 2 HTML code is the same - Example 2 missing the second optional cookies. When the second optional cookies category is added the legend should match the question and read 'Can we use [category] cookies'
- ~Open example on new page links - link is broken on Martin's machine; unless he's in incognito mode~ MW - can't reproduce this problem.
